komga-webui/src/locales/en.json

```diff
@@ -220,53 +220,53 @@
     "year": "year"
   },
   "dashboard": {
-    "keep_reading": "Keep Reading",
-    "on_deck": "On Deck",
-    "recently_added_books": "Recently Added Books",
-    "recently_added_series": "Recently Added Series",
-    "recently_read_books": "Recently Read Books",
-    "recently_released_books": "Recently Released Books",
-    "recently_updated_series": "Recently Updated Series"
+    "keep_reading": "Keep reading",
+    "on_deck": "On deck",
+    "recently_added_books": "Recently added books",
+    "recently_added_series": "Recently added series",
+    "recently_read_books": "Recently read books",
+    "recently_released_books": "Recently released books",
+    "recently_updated_series": "Recently updated series"
```

Comment:
Those are section headers, which in english usually have capitalization for every word, not just the first one.

https://www.webucator.com/article/how-to-capitalize-headings-and-titles/#:~:text=Just%20follow%20these%20simple%20rules,for%2C%20off%2C%20on).